libs/core/src/lib/token/token.component.html

```diff
@@ -2,6 +2,7 @@
     #tokenWrapperElement
     class="fd-token"
     role="option"
+    tabindex="0"
```

Comment:
That's not the expected behaviour. The token should not be accessed by Tab key but with the left/right arrow keys. The Tab should move the focus from one input field to another. Please check the UI5 demo app to see the component behaviour.

Reply:
You need to add `aria-posinset` and  `aria-setsize` as you can add/remove elements which means not all items in a set are present in the DOM all the time. Please check this documentation for more info: https://www.digitala11y.com/aria-posinset-properties/

Reply:
Okay, will check, thanks.

Reply:
Removed `tabindex="0"` which intended to focus on the 1st token in the list if any. Which is not correct behavior.